README.md

```diff
@@ -1 +1,105 @@
+## cryptocurrency-metrics App
+
+## *Microverse React Project*
+
+> This educational project is to build a web application for a project that provides cryptocurrency statistics. The application will allow users to check the prices of coins and all their statistics.
+
+## Learning Objectives 🔖
+- [x] Use React documentation.
+- [x] Use React components.
+- [x] Use React props.
+- [x] Use React Router.
+- [x] Connect React and Redux.
+- [x] Handle events in a React app.
+- [x] Write unit tests with React Testing Library.
+- [x] Use styles in a React app.
+- [x] Use React life cycle methods.
+- [x] Apply React best practices and language style guides in code.
+- [x] Use store, actions and reducers in React.
+- [x] Perform a code review for a team member.
+
+## Built With 🛠️
+
+- [x] Languages: HTML, Css and JavaScript
+- [x] Linters: Stylelint and ESLint
+- [x] Front-end library: React with Redux Toolkit
+- [x] API: SpaceX API
+- [x] Code Editor: VS Code
+
+
+## Live Demo (Not available)
+
+## Getting Started
+
+To get a local copy up and running follow these simple steps.
+
+### Prerequisites
+
+- [x] A web browser like Google Chrome.
+- [x] A code editor like Visual Studio Code with Git and Node.js.
+
+You can check if Git is installed by running the following command in the terminal.
+```
+$ git --version
+```
+
+Likewise for Node.js and npm for package installation.
+```
+$ node --version && npm --version
+```
+
+### Setup
+
+Clone the repository using the GitHub link provided below.
+
+### Install
+
+In the terminal, go to your file directory and run this command.
+
+```
+$ https://github.com/leonard33/cryptocurrency-metrics.git
+```
+
+### Usage
+
+Kindly modify the files as needed.
+
+### Run tests
+
+To check for linters, use the files provided by [Microverse](https://github.com/microverseinc/linters-config). A GitHub action is also set to run during pull request.
+```
+$ npm install
+```
+
+### Deployment(Not available)
+
+
+## Author
+
+👤 **Leonard Kombo** <img src="https://emojis.slackmojis.com/emojis/images/1531849430/4246/blob-sunglasses.gif?1531849430" width="20"/>
+
+- GitHub: [![GitHub Badge](https://img.shields.io/badge/-leonard33-white?logo=GitHub&logoColor=181717&style=plastic)](https://github.com/leonard33)
+
+- Twitter: [![Twitter Badge](https://img.shields.io/badge/-KomboLeonard1-white?logo=Twitter&logoColor=1DA1F2&style=plastic)](https://twitter.com/@LeonardKombo1)
+
+- LinkedIn: [![LinkedIn Badge](https://img.shields.io/badge/-leonardkombo-white?logo=LinkedIn&logoColor=0A66C2&style=plastic)](https://www.linkedin.com/in/leonardkombo/)
+
+- Gmail: [![Gmail Badge](https://img.shields.io/badge/-leonardkombo14-white?logo=Gmail&logoColor=EA4335&style=plastic)](mailto:leonardkombo14@gmail.com)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- [Microverse](https://www.microverse.org/)
+- Code Reviewers
```

Comment:
- [ ] The README contains an "Acknowledgements" section or similar that gives appropriate credit to [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the [author of the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), as required by the [Creative Commons License](https://creativecommons.org/licenses/).